draft-packages/button/KaizenDraft/Button/styles.scss

```diff
@@ -0,0 +1,467 @@
+@import "~@kaizen/component-library/styles/grid";
+@import "~@kaizen/deprecated-component-library-helpers/styles/color";
+@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/typography";
+
+$caButton-border-width: $kz-border-solid-border-width;
+$caButton-focus-border-width: $kz-border-focus-ring-border-width;
+$caButton-height: $ca-grid * 2;
+$caButtonIcon-height: 20px;
+$caButton-formHeight: $ca-grid * 5/3;
+$caButton-verticalPadding: calc(
+  #{$ca-grid / 2} - #{$kz-border-solid-border-width}
+);
+$caButton-verticalPaddingForm: calc(
+  #{$ca-grid / 3} - #{$kz-border-solid-border-width}
+);
+
+// reset user agent syles for button elment type
+@mixin button-reset {
+  appearance: none;
+  background: transparent;
+  border: none;
+  font: inherit;
+  margin: 0;
+  padding: 0;
+  transition: none; // override Murmur global styles :(
+}
+
+%caButtonContainer {
+  display: inline-block;
+}
+
+%caButton {
+  @include button-reset;
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Something for the back burner, but we should probably centralise these text rendering decisions somehow as they don't seem specific to Button - github issue perhaps?

Reply:
Agreed. I'll make an issue 👍

Reply:
Btw they should probably go in the front end ops CSS reset, come to think of it.